common/src/thrift/worker.thrift

```diff
@@ -34,11 +39,11 @@ service WorkerService extends common.TachyonService {
     throws (1: exception.TachyonTException e, 2: exception.ThriftIOException ioe)
 
   /**
-   * Locks the file in Tachyon's space while the session is reading it, and the path of the block file
-   * locked will be returned, if the block file is not found, FileDoesNotExistException will be
-   * thrown.
+   * Locks the file in Tachyon's space while the session is reading it. If lock succeeds, the path of 
+   * the block's file along with the internal lock id of locked block will be returned. If the block's file 
+   * is not found, FileDoesNotExistException will be thrown.
    */
-  string lockBlock( /** the id of the block being accessed */ 1: i64 blockId,
+  LockBlockResult lockBlock( /** the id of the block being accessed */ 1: i64 blockId,
```

Comment:
This changes the API for the worker. @jsimsa , when was the API version checking added? If it was after 0.8, we don't have to change the API version.